apps/src/storage/firebaseStorage.js

```diff
@@ -502,6 +504,32 @@ FirebaseStorage.copyStaticTable = function(tableName, onSuccess, onError) {
     .then(onSuccess, onError);
 };
 
+function enforceUniqueTableNames(tableName) {
+  let projectRef = getProjectDatabase();
+  return Promise.all([
+    projectRef
+      .child(`counters/tables/${tableName}`)
```

Comment:
I noticed 'counters/tables' in a lot of places. Would it make sense to move it to a variable?

Reply:
Done